.github/workflows/release-official.yml

```diff
@@ -37,5 +37,5 @@ jobs:
         with:
           context: "{{defaultContext}}:explorer/ui"
           push: true
-          tags: omniops/explorer-ui:main,omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }},omniops/explorer-ui:${GITHUB_REF##*/}
+          tags: omniops/explorer-ui:main,omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }},omniops/explorer-ui:${{ github.ref_name }}
```

Comment:
Would this continue to work if we deploy staging env?

Reply:
It's fine to release the `ref_name` but we can leave the short git sha. Tags are cheap - let's push both.